### ISSUE TITLE
Add analyze action to .travis.yml xcodebuild calls

### ISSUE THREAD

**Issue by benasher44** [Instagram/IGListKit#52]
## New issue checklist

- [x] I have reviewed the [`README`](https://github.com/Instagram/IGListKit/blob/master/README.md) and [documentation](http://instagram.github.io/IGListKit)
- [x] I have searched [existing issues](https://github.com/Instagram/IGListKit/issues) and this is not a duplicate

### Steps

- Fix remaining analyzer warnings.
- Add the analyze action to `xcodebuild` calls in .travis.yml, and ensure that builds that fail the analyzer fail the build.

**Comment by benasher44** [comment]
Fixed on analyzer issue in #51

**Comment by rnystrom** [comment]
Nice, this sounds really helpful. I imagine once the build is working we need to squish all the analyzer warnings before adding it to the `.travis.yml` file, so builds are green? Or add it first and try to fix the build?

**Comment by benasher44** [comment]
Yep! On master when I checked yesterday, there were 2. There's only the one left after #51. The other one seemed non-trivial to fix.

**Comment by benasher44** [comment]
The final analyzer warning was fixed by #69, which is a pretty awesome patch btw @valeriyvan! This ticket is unblocked now.
